api/apps/articles/test/model.coffee

```diff
@@ -316,23 +316,19 @@ describe 'Article', ->
               article.title.should.equal 'Foo Bar Baz'
               done()
 
-    it 'changes the slug if admin updates it', (done) ->
-      fabricate 'users', { name: 'Molly'}, (err, @user) ->
+    it 'appends the date to an article URL when its slug already exists', (done) ->
+      fabricate 'articles', {
+        slugs: ['craig-spaeth-heyo']
+        }, ->
```

Comment:
Small, but typically better to de-indent this line. Braces for hashes often should continue on the line they're declared e.g.

````javascript
{
  foo: 'bar',
  baz: 'qux'
}
foo = {
  baz: 'bar'
}
doThingToThing('foo', {
  foo: 'bar'
});
````